pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -254,8 +254,30 @@ type RolloutPause struct {
 	Duration *int32 `json:"duration,omitempty"`
 }
 
+// PauseReason reasons that the rollout can pause
+type PauseReason string
+
+const (
+	// InconclusiveAnalysisRun pauses rollout when rollout has an inconclusive analysis run
+	InconclusiveAnalysisRun PauseReason = "InconclusiveAnalysisRun"
+	// CanaryPauseStep pause rollout for canary pause step
+	CanaryPauseStep PauseReason = "CanaryPauseStep"
+	// BlueGreenPause pause rollout before promoting rollout
+	BlueGreenPause PauseReason = "BlueGreenPause"
```

Comment:
can we prefix the reasons, also get rid of "Run" from InconclusiveAnalysisRun
```
PauseReasonInconclusiveAnalysis PauseReason = "InconclusiveAnalysis"
PauseReasonCanaryPauseStep = "CanaryPauseStep"
PauseReasonBlueGreenPause = "BlueGreenPause"
```

Reply:
👍

Reply:
You forgot:
> also get rid of "Run" from InconclusiveAnalysisRun